tests/e2e/Event.test.ts

```diff
@@ -68,18 +74,99 @@ describe('e2e', () => {
     })
 
     describe('#getAll()', async () => {
+      let updateTxId
+      let tlTransferTxId
+      let depositTxId
+      let withdrawTxId
+      let transferTxId
+      let fillTxId
+      let cancelTxId
+
       before(async () => {
-        const [ txObj1, txObj2 ] = await Promise.all([
+        // CurrencyNetwork events
+        const [ updateTx1, updateTx2 ] = await Promise.all([
           tl1.trustline.prepareUpdate(network2.address, user2.address, 1000, 500),
           tl2.trustline.prepareUpdate(network2.address, user1.address, 500, 1000)
         ])
-        await Promise.all([
-          tl1.trustline.confirm(txObj1.rawTx),
-          tl2.trustline.confirm(txObj2.rawTx)
+        const txIds = await Promise.all([
+          tl1.trustline.confirm(updateTx1.rawTx),
+          tl2.trustline.confirm(updateTx2.rawTx)
+        ])
+        updateTxId = txIds[0]
+        await wait()
+        const tlTransferTx = await tl1.payment.prepare(network2.address, user2.address, 1)
+        tlTransferTxId = await tl1.payment.confirm(tlTransferTx.rawTx)
+        await wait()
+        // Token events
+        const depositTx = await tl1.ethWrapper.prepDeposit(ethWrapperAddress, 0.005)
+        depositTxId = await tl1.ethWrapper.confirm(depositTx.rawTx)
+        await wait()
+        const withdrawTx = await tl1.ethWrapper.prepWithdraw(ethWrapperAddress, 0.001)
+        withdrawTxId = await tl1.ethWrapper.confirm(withdrawTx.rawTx)
+        await wait()
+        const transferTx = await tl1.ethWrapper.prepTransfer(ethWrapperAddress, tl2.user.address, 0.002)
+        transferTxId = await tl1.ethWrapper.confirm(transferTx.rawTx)
+        await wait()
+        // Exchange events
+        const order = await tl1.exchange.makeOrder(
+          exchangeAddress,
+          network1.address,
+          network2.address,
+          3,
+          3
+        )
+        const [ fillTx, cancelTx ] = await Promise.all([
+          tl2.exchange.prepTakeOrder(order, 1),
+          tl1.exchange.prepCancelOrder(order, 1)
+        ])
+        const exTxIds = await Promise.all([
+          tl2.exchange.confirm(fillTx.rawTx),
+          tl1.exchange.confirm(cancelTx.rawTx)
         ])
+        fillTxId = exTxIds[0]
+        cancelTxId = exTxIds[1]
         await wait()
       })
 
+      it('should return all events', async () => {
```

Comment:
Please also add a test that test the event more and that the keys are right, something like https://github.com/trustlines-network/clientlib/blob/3dc16342e6e9dfbbbd7b51c294438d71d8fa7eb4/tests/e2e/Event.test.ts#L252. 
Does not have to be here, but somewhere.

Reply:
alright done